nixos/modules/services/networking/wireguard.nix

```diff
@@ -160,6 +160,14 @@ let
         interval of 25 seconds; however, most users will not need this.'';
       };
 
+      table = mkOption {
+        default = "main";
+        type = types.str;
+        description = ''The kernel routing table to add this peer's associated
```

Comment:
I'd probably want to keep it at the per-peer level -- it enables a few more use cases, without significant drawbacks: If I'm only configuring a single peer on each interface, then I'll only need to set `table` once per interface definition anyway -- just as if it were a per-interface option.

I think the "multiple interface, single peer" model is simpler and more flexible, but other people will most probably have different requirements. For example, "ingress filtering for free" is a significant advantage for ease of use.

> Most specific match wins.

So what if multiple peers have the same CIDR range configured as `AllowedIPs`? Will packets be forwarded to just one peer? Or all of them in a round-robin fashion? If the latter, how can one deal with TCP packet reordering?